woocommerce/woocommerce-gutenberg-products-block
# Add product query support for Product Tag list

Adds Product Query support for the atomic Product Tag List block.

On the client side, when the Product Tag List block is used within the product query block, the markup will be rendered on the server side - No javascript related to the Product Tag List block will be rendered.

Fixes #7634 

### Testing
1. Checkout this branch and run npm start
2. Be sure that you have the latest version of the Gutenberg feature plugin installed and activated
3. Create a post/page and add the Product Query block.
4. Add Product Tag List block to Post Template of product query block as shown in video below:

https://user-images.githubusercontent.com/16707866/205036917-aee8bc78-9bc3-4056-89f7-9e55ab3db06a.mov

5. Now save the post & confirm that: 
    1. No JS related to the Product Tag List block is loaded (search in your network tab for product-tag-list-frontend.js). As you can see in the screenshot below, product-tag-list-frontend.js is showing. **This shouldn't be visible in the network tab.**
![image](https://user-images.githubusercontent.com/16707866/205037195-9c8945cc-c577-475c-8060-575603693be3.png)

    2. UI of block should be identical in Frontend & Editor.
    3. The settings are reflected correctly (eg: custom style). Try changing custom style from the sidebar.
    4. ADDITIONAL CSS CLASS(ES)(available in advanced toggle in sidebar) should be added to the container div


#### Automated Tests


### WooCommerce Visibility
